content/templates/implementations.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Implementations
+date: "2024-01-4T12:01:05.284Z"
+description: "A curated list of awesome things related to Lottie"
+image: null
+---
+
+## Resources
+
+- <a href="https://lottiefiles.com/what-is-lottie" target="_blank">What is Lottie</a> by @lottiefiles
+- <a href="https://lottiefiles.github.io/lottie-docs/" target="_blank">Lottie Docs</a> by @lottiefiles
+- <a href="https://airbnb.io/lottie/#/" target="_blank">Lottie Docs</a> by @airbnb
+
+<br/>
+
+## Creation Tools
+
+<br/>
+
+##### Plugins
+
+- <a href="https://lottiefiles.com/ae" target="_blank">LottieFiles AE Plugin</a> for Adobe After Effects
+- <a href="https://aescripts.com/bodymovin/" target="_blank">Bodymovin</a> for Adobe After Effects
+
+##### For Design Tools
+
+- <a href="https://store.framer.com/package/airbnb/lottie" target="_blank">FramerX Lottie Package</a>
+- <a href="https://www.protopie.io/learn/basics/" target="_blank">Protopie Lottie Layer</a>
+- <a href="https://www.figma.com/community/plugin/1307008445393559148" target="_blank">Figma to Lottie</a> by @lottielab
+- <a href="https://university.webflow.com/courses/after-effects-lottie" target="_blank">Lottie Support for Webflow </a>
+- <a href="https://www.figma.com/community/plugin/809860933081065308" target="_blank">Figma to Lottie</a> by @lottiefiles
+
+##### Apps
+
+- <a href="https://www.keyshapeapp.com/" target="_blank">Keyshape for Mac </a>
+- <a href="https://www.haikuforteams.com/animator/" target="_blank">Animator</a> by @haikuforteams
+- <a href="https://glaxnimate.mattbas.org/ " target="_blank">Glaxnimate </a> by @mbasaglia
+
+<br/>
+
+## Libraries
+
+<br/>
+
+##### Web
+
+- <a href="https://github.com/airbnb/lottie-web" target="_blank">lottie-web</a> - official
+- <a href="https://github.com/LottieFiles/lottie-player" target="_blank">lottie-player</a> - by @lottiefiles
+- <a href="https://github.com/LottieFiles/lottie-react" target="_blank">lottie-react</a> - by @lottiefiles
```

Comment:
Since we have two things named lottie-react, would it make sense to name this `react-lottie-player` to match the npm package it is published under and better distinguish the two??